src/peer_manager.rs

```diff
@@ -43,7 +43,8 @@ use xor_name::XorName;
 const JOINING_NODE_TIMEOUT_SECS: u64 = 900;
 /// Time (in seconds) after which the connection to a peer is considered failed.
 const CONNECTING_PEER_TIMEOUT_SECS: u64 = 90;
-/// Time (in seconds) the node waits for a peer to become valid once connected.
+/// Time (in seconds) the node waits for a peer to become valid once connected or bootstrapped to
+/// it.
```

Comment:
Change force pushed (nothing altered except this comment to the suggested text).